src/devices/Buzzer/Buzzer.csproj

```diff
@@ -9,11 +9,13 @@
   <ItemGroup>
     <Compile Include="Buzzer.cs" />
     <None Include="README.md" />
-    <PackageReference Include="System.Device.Gpio" Version="$(SystemDeviceGpioPackageVersion)" />
+    <ProjectReference Include="$(MSBuildThisFileDirectory)../../System.Device.Gpio/System.Device.Gpio.csproj">
```

Comment:
why is MSBuildThisFileDirectory needed?
Does `<AdditionalProperties>RuntimeIdentifier=linux</AdditionalProperties>` have to be passed? What about Windows IoT Core?

Reply:
Additional properties is required, and it will work even with Windows IoT Core. Basically this means that the surface area that we want to use is the linux one, as the Windows one has some extra base classes for its implementation. MSBuildThisFileDirectory is not strictly required, but it is convention to try to have full paths instead of relative ones. We do this in the rest of our repos as well when possible.